systemtest/src/main/java/io/strimzi/systemtest/resources/specific/HelmResource.java

```diff
@@ -126,7 +126,7 @@ private void clusterOperator(long operationTimeout, long reconciliationInterval)
                 entry("logLevelOverride", Environment.STRIMZI_LOG_LEVEL),
                 entry("fullReconciliationIntervalMs", Long.toString(reconciliationInterval)),
                 entry("operationTimeoutMs", Long.toString(operationTimeout)),
-                entry("featureGates", Environment.STRIMZI_FEATURE_GATES),
+                entry("featureGates", Environment.STRIMZI_FEATURE_GATES.replaceAll(",", "\\\\,")),
```

Comment:
Do we want some explanatory comment about why it needs to be escaped like this?

Reply:
Good idea, I would put it there as it's easy to miss.